File: 001/c/solution.c
#include <stdio.h>
#include "multiple.h"

int solve(int p);

#ifndef TESTING
int main () {
  int p;
  scanf("%d", &p);
  printf("%d\n", solve(p));
  return 0;
}
#endif

// [001/explanation.tex]
int solve(int p) {
  return multiples_under(p, 3)
       + multiples_under(p, 5)
       - multiples_under(p, 15);
}

File: 001/go/solution.go
package main

import "fmt"
import "pe"

func main() {
  var p int
  fmt.Scanln(&p)
  fmt.Println(solve(p));
}

func solve(p int) int {
  return pe.MultiplesUnder(p, 3) + pe.MultiplesUnder(p, 5) - pe.MultiplesUnder(p, 15)
}
